src/renderer/store/modules/settings.js

```diff
@@ -232,6 +232,7 @@ const state = {
   listType: 'grid',
   maxVideoPlaybackRate: 3,
   onlyShowLatestFromChannel: false,
+  openLinksInNewWindow: false,
```

Comment:
This is not for any link shown inside FT
[Electron doc calls it deep link](https://www.electronjs.org/docs/latest/tutorial/launch-app-from-url-in-another-app#overview) if I read that right
```suggestion
  openDeepLinksInNewWindow: false,
```